src/Parser/Shaman/Restoration/Modules/Talents/Downpour.js

```diff
@@ -49,12 +55,43 @@ class Downpour extends Analyzer {
 
     if(event.amount) {
       this.downpourHits += 1;
+      this.downpourHitsSum += 1;
     }
 
     this.downpourTimestamp = event.timestamp;
     this.healing += event.amount;
   }
 
+  statistic() {
+    const downpour = this.abilityTracker.getAbility(SPELLS.DOWNPOUR_TALENT.id);
+
+    const downpourCasts = downpour.casts || 0;
+    if(!downpourCasts) {
+      return null;
+    }
+    // downpourHits are all hits and downpourHitsSum are only the ones with effective healing done
+    const downpourHits = downpour.healingHits || 0;
+    const downpourAverageHits = (this.downpourHitsSum) / downpourCasts;
+    const downpourAverageOverhealedHits = (downpourHits - this.downpourHitsSum) / downpourCasts;
+    const downpourAverageCooldown = 5 + (this.downpourHitsSum / downpourCasts * 5);
+
+    return (
+      <StatisticBox
+        icon={<SpellIcon id={SPELLS.DOWNPOUR_TALENT.id} />}
+        value={`${downpourAverageCooldown.toFixed(1)} seconds`} 
+        label={(
+          <dfn data-tip={`
+            You cast a total of ${downpourCasts} Downpours, which on average hit ${(downpourAverageHits + downpourAverageOverhealedHits).toFixed(1)} out of 6 targets. <br /> 
+            Of those hits, ${downpourAverageHits.toFixed(1)} had effective healing and increased the cooldown.
+          `}>
+            Average Downpour cooldown
```

Comment:
Would it be more intuitive to show average number of effective hits as the primary value instead? Maybe it's just cause I don't play shaman but to me the average cooldown doesn't say much about how effective the usage was (other than higher -> better)

Reply:
My issue with Downpour is that higher isn't necessarily better, apart from mana usage and the small base cooldown punishment. Just showing raw hits wouldn't tell you the full story as **effective** hits is what impacts the cooldown time. Showing only effective hits could be confusing as all others (chain heal, healing rain, gift of the queen) show raw hits, and showing both is hard to read.
Having it be the average cooldown time with more information on hover is the best compromise i was able to come up with.